cypress/e2e/tests.cy.js

```diff
@@ -0,0 +1,108 @@
+import userResponse from "../fixtures/userResponse.json";
+import ObjectPage from "../pages/objectsPage.cy";
+
+const objectPage = new ObjectPage();
+
+describe("App example - Cypress workshop", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  context("Mock user response", () => {
```

Comment:
we don't need here context - all tests has the same prerequisite (and actually- it's our internal agreement to add to context info about what do we mock, so it's not wide spreaded)

Reply:
I added also the second context with API error:
``````
 context("Mock response with status 400", () => {
    it.only("Validate API respose: 400", () => {
      cy.intercept("GET", "https://randomuser.me/api/", (req) => {
        req.reply({
          statusCode: 400,
          fixture: "userResponse.json",
        });
      }).as("error");

      cy.get("button").contains("Generate new user").click();
      cy.wait("@error");

      // dev should catch error  - after click on Generate new user we don't get any info about 400, after click nothing changed
    });
  });
```

it will be added in next commit

Reply:
so maybe separate file with tests that validates error?